src/moditect/module-info.java

```diff
@@ -0,0 +1,11 @@
+module com.fasterxml.jackson.core {
+	exports com.fasterxml.jackson.core;
+
+	exports com.fasterxml.jackson.core.type to com.fasterxml.jackson.databind, com.jwebmp.core, com.fasterxml.jackson.datatype.jsr310;
+	exports com.fasterxml.jackson.core.io to com.fasterxml.jackson.databind, com.fasterxml.jackson.module.afterburner, com.fasterxml.jackson.datatype.jdk8;
+	exports com.fasterxml.jackson.core.util to com.fasterxml.jackson.databind, com.fasterxml.jackson.module.paranamer, com.fasterxml.jackson.module.parameternames, com.fasterxml.jackson.module.osgi, com.fasterxml.jackson.module.mrbean, com.fasterxml.jackson.module.jaxb, com.fasterxml.jackson.module.afterburner, com.fasterxml.jackson.datatype.jsr310, com.fasterxml.jackson.datatype.jdk8, com.fasterxml.jackson.module.guice;
```

Comment:
Some of the types needed widely (about half), so needs to be exported to all. May want to refactor for 3.0 to hide some implementations.

Reply:
I think doing a wide export (but not an opens all) will work, and then in 3 go "strict"? 
The big things here are just that the module is named so it works with jlink and transitive requires clauses, not really how it is exported (I guess), I tried to get as close to good with what I know, but thinking about, and how it currently is looking, a loose jpms module for 2.x compat rather than a strict may be the way to go